docs/docs/reference-react-dom-node-stream.md

```diff
@@ -0,0 +1,42 @@
+---
+id: react-dom-node-stream
+title: ReactDOMNodeStream
+layout: docs
+category: Reference
+permalink: docs/react-dom-node-stream.html
+---
+
+If you use ES6 with npm, you can write `import ReactDOMNodeStream from 'react-dom/node-stream'`. If you use ES5 with npm, you can write `var ReactDOMNodeStream = require('react-dom/node-stream')`.
+
+Unlike other packages in React, `ReactDOMNodeStream` depends on a package (`stream`) that is available in Node.js but not in the browser. For this reason, there is no `<script>` tag version of `ReactDOMNodeStream`; it is only provided as a Node.js module.
+
+## Overview
+
+The `ReactDOMNodeStream` class allows you to render your components in Node.js and stream the resulting markup.
+
+ - [`renderToStream()`](#rendertostream)
+ - [`renderToStaticStream()`](#rendertostaticstream)
+
+* * *
+
+## Reference
+
+### `renderToStream()`
+
+```javascript
+ReactDOMNodeStream.renderToStream(element)
+```
+
+Render a React element to its initial HTML. This should only be used in Node.js; it will not work in the browser, since the browser does not support Node.js streams. React will return a [https://nodejs.org/api/stream.html#stream_readable_streams](Readable stream) that outputs an HTML string. The HTML output by this stream should be character-for-character equivalent to the output of [https://facebook.github.io/react/docs/react-dom-server.html#rendertostring](ReactDOMServer.renderToString). You can use this method to generate HTML on the server and send the markup down on the initial request for faster page loads and to allow search engines to crawl your pages for SEO purposes.
```

Comment:
>  The HTML output by this stream should be character-for-character equivalent to the output...

Phrasing is a little awkward, could we convey the same idea without "character-for-character" and "should"?

> The HTML output by this stream will be exactly equal to what `ReactDOMServer.renderToString` would return.